backend/app/lm/handlers.py

```diff
@@ -167,8 +170,14 @@ async def set_content(message=message):
                 ses, usr, event(ses, arena_request_event.id), judge_score
             )
             evaluate.delay(judge_score.then(judge_score_event))
-        return chat_completion_response
-
+        return chat_completion_with_real_entities
+
+def replace_back(text: str, mapping: list[dict[str,str]]) -> str:
+        for fake_entity, real_entity in mapping.items():
+            print("text",text)
+            text = text.replace(real_entity, fake_entity)
```

Comment:
Your naming is ambiguous here: str.replace(a,b) replace the string a with b. It seems to me that you should be doing the opposite?